app/javascript/app/components/my-climate-watch/viz-creator/components/charts/legend/legend-styles.scss

```diff
@@ -19,14 +19,19 @@
   box-shadow: none;
 }
 
-.legendLogo {
+.legendLogoContainer {
   display: flex;
   flex-direction: column;
+  align-items: center;
   color: $dark-gray;
-  max-width: 150px;
+  min-width: 110px;
   font-size: $font-size-sm;
 }
 
 .legendLogoTitle {
   padding-bottom: 10px;
 }
+
+.legendLogo {
+  width: 100px;
```

Comment:
We could use 115px to be perfectly aligned to the 'Data provided by:' text. Just being picky 😝but its ok. It's true that some of them seem bigger and are easier to read